<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">fb2cut</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="_7">
        <item>
         <widget class="QLabel" name="labelInputFiles">
          <property name="text">
           <string>Input archive file or wildcard</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelInputFilesFound">
          <property name="accessibleName">
           <string>Files found: %1</string>
          </property>
          <property name="text">
           <string notr="true">Files found:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_3">
        <item>
         <widget class="QLineEdit" name="editInputFiles"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnInputFiles">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="resources/fb2cut.qrc">
            <normaloff>:/icons/add_file.png</normaloff>:/icons/add_file.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelDstFolder">
        <property name="text">
         <string>Destination folder</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_4">
        <item>
         <widget class="QLineEdit" name="editDstFolder">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDstFolder">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="resources/fb2cut.qrc">
            <normaloff>:/icons/add_folder.png</normaloff>:/icons/add_folder.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>6</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_5">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionInputFilesNotFound">
   <property name="icon">
    <iconset resource="resources/fb2cut.qrc">
     <normaloff>:/icons/exclamation.png</normaloff>:/icons/exclamation.png</iconset>
   </property>
   <property name="text">
    <string>InputFiles</string>
   </property>
   <property name="toolTip">
    <string>No input archives found</string>
   </property>
  </action>
  <action name="actionDstFolderNotExists">
   <property name="icon">
    <iconset resource="resources/fb2cut.qrc">
     <normaloff>:/icons/exclamation.png</normaloff>:/icons/exclamation.png</iconset>
   </property>
   <property name="text">
    <string>DstFolder</string>
   </property>
   <property name="toolTip">
    <string>Destination folder not exists</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/fb2cut.qrc"/>
 </resources>
 <connections/>
</ui>
